C/services/south/south.cpp

```diff
@@ -202,8 +202,11 @@ void SouthService::start(string& coreAddress, unsigned short corePort)
 
 		try {
 			m_pollInterval = 500;
+			m_readingsPerSec = 1;
 			if (m_config.itemExists("pollInterval"))
```

Comment:
We can probably ditch the pollInterval stuff completely and just have the readingsPerSec for today

Reply:
Done.